src/util/metrics/threads_linux.rs

```diff
@@ -220,11 +200,33 @@ fn get_thread_name(tid: pid_t, stat: &str) -> Result<(String, usize)> {
     )))
 }
 
+// get thread name and the index of the last character(including ')').
+fn sanitize_thread_name(tid: pid_t, raw: &str) -> String {
```

Comment:
Is that necessary to extract this function? IMO, get_thread_name() returns a sanitized name is more intuitive

Reply:
sanitized will replace `-` to `_`, which is not friendly in other places. For example, gRPC threads are named with prefix `grpc-thread`, and if `get_thread_name` returns a same strings, we can use a const value to test it. It's good for avoid some mistakes.